releases/v0.3.0.md

```diff
@@ -1,12 +1,20 @@
 # Release Notes for v0.3.0
-Release Date: TBD
+Release Date: 1/19/23
 
-Code Freeze: TBD
+Code Freeze: 1/13/23
 
 Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential Containers
 
 ## What's new
 - Support for pulling images from authenticated container registries. See [design info](https://github.com/confidential-containers/image-rs/blob/main/docs/image_auth.md).
+- Significantly reduced resource requirements for image pulling
+- Attestation support for AMD SEV-ES
+- `kata-qemu-tdx` supports and has been tested with Verdictd
+- Support for `get_resource` endpoint with SEV(-ES)
+- Enabled cosign signature support in enclave-cc / SGX
+- SEV attestation bug fixes
+- Measured rootfs now works with `kata-clh`, `kata-qemu`, `kata-clh-tdx`, and `kata-qemu-tdx` runtime classes.
+- Enhanced docs, config, CI pipeline and test coverage for enclave-cc / SGX
```

Comment:
@fitzthum One more remark on s390x: Could you add this to the `What's new`:

`IBM zSystems / LinuxONE (s390x) enablement and CI verification on non-TEE environments`

Also in section `Hardware Support`, could you replace `IBM Z` with:

`IBM Secure Execution (SE) on IBM zSystems & LinuxONE`